foca/models/config.py

```diff
@@ -4,7 +4,10 @@
 import os
 from typing import (Dict, List, Optional, Tuple)
 
-from pydantic import (BaseModel, Field, validator)  # pylint: disable=E0611
+from pydantic import (BaseSettings,
+                      Field,
```

Comment:
Please stick to the indentation style that's used elsewhere. I think the style you're using is easy to read, but it's very wasteful and quickly becomes tricky when lines are long.

Style used is, e.g.:

```py
my_dict = {
    "my_key": "my_value",
}
```

And please try to always put a comma after the last value. Big advantage of Python vs, e.g., JS. Much easier to maintain.